Sim/SimG4Components/src/SimG4DD4hepDetector.h

```diff
@@ -32,7 +32,7 @@ class G4DD4hepDetector: public GaudiTool, virtual public IG4DetectorConstruction
   /** Get the initilization of the geometry.
    *  @return pointer to G4VUserDetectorConstruction
    */
-  virtual G4VUserDetectorConstruction* getDetectorConstruction();
+  virtual G4VUserDetectorConstruction* detectorConstruction();
```

Comment:
how is ownership handled here? I assume also the ownership is transferred to the caller?